scripts/startup/tmux_launch.sh

```diff
@@ -34,14 +46,20 @@ echo "PANDIR: ${PANDIR}"
 echo "PANLOG: ${PANLOG}"
 echo "POCS: ${POCS}"
 echo "PAWS: ${PAWS}"
-echo "PIAA: ${PIAA}"
-echo "MATPLOTLIBRC: ${MATPLOTLIBRC}"
+
 # $- expands to the current option flags as specified upon invocation, by the
 # set built-in command, or those set by the shell itself (such as the -i).
-echo '$-:' "$-"
+echo 'Shell options ($-):' "$-"
+
+# These two environment variables don't have to be set, but let's record
+# their values as they may impact things.
+echo "PIAA: ${PIAA}"
+echo "MATPLOTLIBRC: ${MATPLOTLIBRC}"
```

Comment:
We move the file into the base of $POCS and are no longer setting this so is probably best to remove.

Reply:
Done.